### ISSUE TITLE
Text link "without surrounding text" is not accessible.

### ISSUE THREAD

**Issue by weboverhauls** [18F/web-design-standards#812]
Under [Visual style/Links](https://playbook.cio.gov/designstandards/visual-style/#links), the example "This is a link without surrounding text" is not fully accessible since it relies on color alone to denote that it's a link (which is the 3rd bullet under Documentation/Accessibility in this section).

Suggestion 1: return the underline.

Suggestion 2: provide more visual affordance that the standalone text is a link such as adding an icon.

WCAG references: 
http://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-without-color
http://www.w3.org/TR/WCAG20-TECHS/F73.html

**Comment by carodew** [comment]
Thanks! You raise a good point and we're going to take a look at this.

**Comment by mollieru** [comment]
@angel hey, can you take a look at this? feel free to ping other members of the team to help think through.

**Comment by colinpmacarthur** [comment]
@weboverhauls: Thanks again for submitting this issue. We totally agree that links shouldn't use color alone to identify themselves. There are several different ways to do that -- we're going to think about the right one to suggest.

**Comment by carodew** [comment]
As we address this we'll need to update our guidance here: https://playbook.cio.gov/designstandards/visual-style/#links

**Comment by ericadeahl** [comment]
@maya I recommend we use the standard link style with the underline, and not distinguish between a link with or without surrounding text. Thoughts @bradnunnally ?

If we do that, we should simply remove all references to "This is a link without surrounding text." on the site, in the code, and in the design files.

One additional issue in the links section: the design file includes an icon to indicate a link that will open in a new tab:

<img width="315" alt="screen shot 2016-05-31 at 3 46 31 pm" src="https://cloud.githubusercontent.com/assets/5911280/15693022/f4d9771e-2746-11e6-81d9-9b82a5f11eb5.png">

but the icon is missing on the Standards website:

<img width="290" alt="screen shot 2016-05-31 at 3 46 41 pm" src="https://cloud.githubusercontent.com/assets/5911280/15693029/fa84f562-2746-11e6-89e2-2fd2140929b4.png">

When we make this update, could we also add the new tab icon to the site?

**Comment by maya** [comment]
@ericadeahl that makes sense to me. 

FYI the second issue about the external link icon is documented here: https://github.com/18F/web-design-standards/issues/575 so we can followup there.

**Comment by ericadeahl** [comment]
@maya Oops cool, thanks!

**Comment by bradnunnally** [comment]
@ericadeahl I agree with your thoughts. 👍

**Comment by joshbruce** [comment]
See #1215

**Comment by leannegunn** [comment]
This should also be removed from the designer files.

**Comment by ericadeahl** [comment]
@leannegunn Thanks for your note! This has been updated in the latest version of our Sketch, AI, and EPS files (and Omnigraffle update to come.)
